Composer/packages/electron-server/src/main.ts

```diff
@@ -310,7 +310,6 @@ async function run() {
   app.on('before-quit', () => {
     const mainWindow = ElectronWindow.getInstance().browserWindow;
     mainWindow?.webContents.send('session-update', 'session-ended');
-    mainWindow?.webContents.send('conversation-cleanup');
```

Comment:
what is this change for?

Reply:
This is actually a remnant from the previous webchat implementation to cleanup all the sockets when electron is closed. It is not required as i close the socket using the destructor in the useEffect (WebchatPanel.tsx). This can be safely removed.